package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "matt-item-summary",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "seed": "node ./server/seed.js"
```

Comment:
I would recommend alphabetizing key-value pairs in objects unless you have a compelling reason to not do so.

Reply:
Sounds good, I was just adding them all will-nilly but I can see how that's inconvenient as the package.json file grows in complexity